Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM bitwalker/alpine-erlang
```

Comment:
... I also had to put the erts bin folder to the $PATH. 
checkout my [Dockerfile](https://gist.github.com/webdeb/668b67be2b5d232b1b0bbbc5b592c99d) , its actually based on bitwalkers work, thank you btw!
cc @bitwalker